bundle/Core/Search/Solr/Query/Common/SortClauseVisitor/ContentTypeId.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Netgen\TagsBundle\Core\Search\Solr\Query\Common\SortClauseVisitor;
+
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use EzSystems\EzPlatformSolrSearchEngine\Query\SortClauseVisitor;
+use Netgen\TagsBundle\API\Repository\Values\Content\Query\SortClause\ContentTypeId as ContentTypeIdClause;
+
+/**
+ * Visits the sortClause tree into a Solr query.
+ */
+class ContentTypeId extends SortClauseVisitor
+{
+    /**
+     * Check if visitor is applicable to current sortClause.
+     *
+     * @param SortClause $sortClause
```

Comment:
FQCN should be used in PHPDoc `@param` tag, here and in method below.

Reply:
Fixed in https://github.com/netgen/TagsBundle/pull/98/commits/05f7dad27c86fbfdc247c7f08f2966bc63b3a183.